### ISSUE TITLE
Imports in Webpack

### ISSUE THREAD

**Issue by jhuntoo** [angular/in-memory-web-api#3]
Using https://github.com/johnpapa/event-view as a guide, I've tried implementing this in my project which uses webpack. However the import statement doesn't work
```
import {
  InMemoryBackendConfig,
  InMemoryBackendService,
  SEED_DATA
} from 'angular2-in-memory-web-api/core';
```
**Console Output**
`bootstrap d05bac4…:50 Uncaught ReferenceError: System is not defined`

The offending line is https://github.com/angular/in-memory-web-api/blob/master/core.js#L1

The workaround for webpack is just to use the import `from 'angular2-in-memory-web-api/src/core'`.

It might be worth changing this to be consistent with most other ng2/typescript libraries.

**Comment by wardbell** [comment]
@foxandxss?

**Comment by Foxandxss** [comment]
So the problem in here is that Webpack 1 doesn't talk "System.js" (Webpack 2 will).

The transpiled source won't work with Webpack so as @jhuntoo said, the workaround is to directly load from the source (which is not really a bad idea).

What the other libraries are doing is just using commonjs and that will work with any solution.

**Comment by wardbell** [comment]
Fixed by PR #5
